hadoop-common-project/hadoop-common/src/main/java/io/hops/security/UsersGroupsCache.java

```diff
@@ -583,7 +583,7 @@ void addUserToGroups(String user,
     Integer userId = null;
```

Comment:
The functions addUserToGroup and addIUserTogroups function are often used to either only add user or only add group but not add the user to the group(s), This is very counter-intuitive and make it difficult for other programmers to understand what function does what. This should be refactored.